README.md

```diff
@@ -62,34 +62,34 @@ and [releases](https://github.com/denoland/deno/releases) for other options.
 
 ### Getting Started
 
-Try running a simple program:
+Try [running a simple program](https://examples.deno.land/hello-world):
 
 ```sh
 deno run https://deno.land/std/examples/welcome.ts
 ```
 
-Or a more complex one:
+Or [setup a simple HTTP server](https://examples.deno.land/http-server):
 
 ```ts
-const listener = Deno.listen({ port: 8000 });
-console.log("http://localhost:8000/");
+import { serve } from "https://deno.land/std@0.182.0/http/server.ts";
 
-for await (const conn of listener) {
-  serve(conn);
+function handler(_req: Request): Response {
+  return new Response("Hello, World!");
 }
 
-async function serve(conn: Deno.Conn) {
-  for await (const { respondWith } of Deno.serveHttp(conn)) {
-    respondWith(new Response("Hello world"));
-  }
-}
+console.log("Listening on http://localhost:8000");
+serve(handler);
```

Comment:
Maybe we should just inline the handler here?

```ts
serve((_req) => {
  return new Response("Hello, World!");
});
```

Reply:
It can be simplified to a single line too

Reply:
```ts
serve(_req => new Response("Hello, World!"));
```

Reply:
The entire snippet can be:
```ts
import { serve } from "https://deno.land/std@0.182.0/http/server.ts";

serve((_req) => new Response("Hello, World!"));
```
1. By default, `serve()` prints `Listening on http://localhost:8000/` so the `console.log()` isn't needed.
2. Running `deno fmt` adds brackets to `_req` parameter.